src/Apps/Auction/Components/AuctionDetails/AuctionDetails.tsx

```diff
@@ -33,6 +33,16 @@ const AuctionDetails: React.FC<AuctionDetailsProps> = ({ sale, me }) => {
   const showCascadingEndTimeIntervalMessage: boolean =
     !!sale.cascadingEndTimeIntervalMinutes && !sale.isClosed
 
+  const getEndTimeIntervalMessage = () => {
+    const { cascadingEndTimeIntervalMinutes } = sale
+
+    if (sale.artworksConnection?.totalCount === 1) {
+      return `Lot close at ${cascadingEndTimeIntervalMinutes}-minute intervals`
+    }
+
+    return `Lots close at ${cascadingEndTimeIntervalMinutes}-minute intervals`
```

Comment:
**thought (non-blocking):** Learned from [the recent i18n presentation](https://artsy.slack.com/archives/C02BC3HEJ/p1668697223204379?thread_ts=1668523213.247079&cid=C02BC3HEJ) that we have access to [pluralization helpers via react-i18next](https://react.i18next.com/misc/using-with-icu-format#plural).